test/glaciers_construction.jl

```diff
@@ -1,10 +1,11 @@
 
 
-function glaciers2D_constructor(; save_refs::Bool = false)
+function glaciers2D_constructor(; save_refs::Bool = true)
```

Comment:
This is VERY important. Never commit files with `save_refs = true`. Otherwise you're basically bypassing all tests. You should only use this to generate the reference data once you're sure the code is working. Then set it to false and commit in order to compare it to a "frozen" reference.